tests/hooks/after_prepare.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+/*
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+*/
+
+var path = require('path');
+var fs = require('fs');
+
+module.exports = function(context) {
+    function main() {
+        // get the file transfer server address from the specified variables
+        var defaultFileTransferServerAddress = getDefaultFileTransferServerAddress(context);
+        var fileTransferServerAddress = getFileTransferServerAddress(context) || defaultFileTransferServerAddress;
+        console.log('Tests will use the following file transfer server address: ' + fileTransferServerAddress);
+
+        // pass it to the tests
+        writeFileTransferOptions(fileTransferServerAddress, context);
+    }
+
+    function getDefaultFileTransferServerAddress(context) {
+        var address = null;
+        var configNodes = context.opts.plugin.pluginInfo._et._root._children;
+
+        for (var node in configNodes) {
+            if (configNodes[node].attrib.name == 'FILETRANSFER_SERVER_ADDRESS') {
+                address = configNodes[node].attrib.default;
+            }
+        }
+
+        return address;
+    }
+
+    function getFileTransferServerAddress(context) {
+        var platformJsonFile = path.join(context.opts.projectRoot, 'plugins', context.opts.platforms[0] + '.json');
```

Comment:
I think you need to use `<project_root>/<platform>/<platform>.json` rather than one in the 'plugins' folder